### PR TITLE
Fix React sample

### PR DESCRIPTION
**Pull request by maartenhvdh**
React sample gave issues, I changed it to my working sample

### Motivation

This fixes React sample in the readme,  issue #18

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,85 +131,83 @@ import { PortableText, PortableTextReactComponents } from "@portabletext/react";
...
-          target={target}
-          rel={value?.rel}
-          title={value?.title}
-          data-new-window={value["data-new-window"]}
-        >
-          {children}
-        </a>
-      );
+const createPortableTextComponents = (linkedItems) => {
+  const portableTextComponents: Partial<PortableTextReactComponents> = {
```
Comment (Simply007) [inline README.md:138]: There is a missing return statement.

File: README.md [hunk 2]
```diff
@@ -131,85 +131,83 @@ import { PortableText, PortableTextReactComponents } from "@portabletext/react";
...
+    const portableText = transformToPortableText(parsedTree);
 
-  const parsedTree = browserParse(props.element.value);
-  const portableText = transformToPortableText(parsedTree);
-
-  return (
-    <PortableText value={portableText} components={portableTextComponents} />
-  );
+    return (
+      <PortableText value={portableText} components={portableTextComponents} />
```
Comment (Simply007) [inline README.md:208]: `portableTextComponents` are note defined - this is the practice by react portable text package..

**Review by Simply007 — COMMENTED** [review]
I am not sure how your use case would work without using resolution logic.

**Comment by pokornyd** [comment]
hello @maartenhvdh. as a part of the v1 release, I adjusted the readme along with the code examples. I've chosen clarity instead of complexity, so they are not copy-paste kind and assume you have pulled the rich text from SDK or otherwise parsed it, something which is mentioned via inline comments in the examples themselves. since there were couple helper methods implemented, the examples reflect that, therefore I don't see the need of merging the adjusted example from this PR. feel free to review the changes and let me know if you're okay with closing this PR or if you want to add something :)

**Comment by pokornyd** [comment]
docs were updated, closing without merging, details in the related issue.

**Pull request closed without merging**